drizzlepac/devutils/comparison_tools/compare_sourcelists.py

```diff
@@ -76,6 +89,8 @@
 ---------------------
 """
 import argparse
+from datetime import datetime
+import os
 import pdb
 import random
```

Comment:
We need to remember to remove `import pdb` from all of our code before it is released.